content/_includes/quotes.macros.njk

```diff
@@ -180,7 +181,8 @@ params:
 {% macro cite(
   data
 ) -%}
-  <p class="byline h-card">---{{ _cite(data) | safe }}</p>
+  <p class="byline h-card">
+    ---{{ _cite(data) | safe }}</p>
```

Comment:
@dvdherron did your formatter leave this closing </p> on this line? I would think it would be on its own so you can easily see where this element ends.

Reply:
strange. @stacyk yes, one of my `njk` formatters made this change. fixed [here](https://github.com/oddbird/oddleventy/pull/193/commits/f5c12c662a956f4b5e15143f251ada8df945d6fd)